src/app/clusters/mode-select-server/mode-select-server.cpp

```diff
@@ -211,6 +238,10 @@ InteractionModel::Status MatterModeSelectClusterServerPreAttributeChangedCallbac
  */
 static InteractionModel::Status verifyModeValue(const EndpointId endpointId, const uint8_t newMode)
 {
+    if (newMode == 0xFF) // This indicates that the new mode is null.
```

Comment:
Please use `NumericAttributeTraits` here instead of hardcoding this.  In particular, `NumericAttributeTraits<uint8_t>::IsNullValue(newMode)`